charts/pxc-db/README.md

```diff
@@ -20,31 +20,36 @@ To install the chart with the `pxc` release name using a dedicated namespace (re
 
 ```sh
 helm repo add percona https://percona.github.io/percona-helm-charts/
-helm install my-db percona/pxc-db --version 1.11.0 --namespace my-namespace
+helm install my-db percona/pxc-db --version 1.12.0 --namespace my-namespace
 ```
 
 The chart can be customized using the following configurable parameters:
 
 | Parameter                       | Description                                                                   | Default                                   |
 | ------------------------------- | ------------------------------------------------------------------------------| ------------------------------------------|
+| `crVersion`                     | Version of the Operator the Custom Resource belongs to                        | `1.12.0`                                  |
+| `ignoreAnnotations`             | Operator will not remove following annotations                                | `[]`                                      |
+| `ignoreLabels`                  | Operator will not remove following labels                                     | `[]`                                      |
 | `pause`                         | Stop PXC Database safely                                                      | `false`                                   |
 | `allowUnsafeConfigurations`     | Allows forbidden configurations like even number of PXC cluster pods          | `false`                                   |
 | `enableCRValidationWebhook`     | Enables or disables schema validation before applying custom resource         | `false`                                   |
 | `initImage`                     | An alternative image for the initial Operator installation                    | `""`                                      |
 | `updateStrategy`                | Regulates the way how PXC Cluster Pods will be updated after setting a new image | `SmartUpdate`                          |
 | `upgradeOptions.versionServiceEndpoint` | Endpoint for actual PXC Versions provider                             | `https://check.percona.com/versions`      |
-| `upgradeOptions.apply`          | PXC image to apply from version service - `recommended`, `latest`, actual version like `8.0.19-10.1` | `8.0-recommended` |
+| `upgradeOptions.apply`          | PXC image to apply from version service - `recommended`, `latest`, actual version like `8.0.19-10.1` | `disabled` |
 | `upgradeOptions.schedule`          | Cron formatted time to execute the update | `"0 4 * * *"` |
```

Comment:
let's fix the version!

Reply:
this is just example for specific old version, doesn't have to be current one